Promo/dataobjects/PromoCartEntry.php

```diff
@@ -5,9 +5,14 @@
  *
  * @copyright 2011-2016 silverorange
  * @license   http://www.opensource.org/licenses/mit-license.html MIT License
+ *
+ * @property ?float $promotion_discount
  */
 class PromoCartEntry extends StoreCartEntry
 {
+    /**
+     * @var float
```

Comment:
Can it be marked nullable here or only at the top of the class?

Reply:
Removed